src/main/java/uk/gov/hmcts/reform/sendletter/tasks/MarkLettersPostedTask.java

```diff
@@ -61,11 +61,11 @@ public void run(LocalTime now) {
                         logger.warn("Report {} contained invalid rows, file not removed.", parsedReport.path);
                     }
                 });
+
+            logger.info("Completed report processing job");
         } else {
-            logger.info("Not processing due to FTP downtime window");
+            logger.info("Not processing '{}' task due to FTP downtime window", Task.MarkLettersPosted);
```

Comment:
Inconsistent with how this job/task is referred to in other logs in this class
(`report processing job` vs `'MarkLettersPosted' task`)